runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -130,6 +131,10 @@ public XWalkContent(Context context, AttributeSet attrs, XWalkViewInternal xwVie
         XWalkPreferencesInternal.load(this);
     }
 
+    public View getContentView() {
```

Comment:
Just because I want to use it like a View only and then hide  the ContentView from XWalkViewInternal.